app/src/main/java/com/infomaniak/mail/data/models/thread/Thread.kt

```diff
@@ -208,6 +229,12 @@ class Thread : RealmObject {
             }
             if (message.hasAttachable) hasAttachable = true
             if (message.isScheduledDraft) numberOfScheduledDrafts++
+
+            updateSnoozeSatesBasedOn(message)
+        }
+
+        duplicates.forEach { duplicate ->
+            updateSnoozeSatesBasedOn(duplicate)
         }
```

Comment:
Do we really need/want to update the Thread based on the duplicated Messages ? 🤔